client/vscode/src/chatActivation.ts

```diff
@@ -49,10 +50,43 @@ export function registerChat(languageClient: LanguageClient, extensionUri: Uri)
                     .sendRequest(chatRequestType, Object.assign(message.params, { partialResultToken }))
                     .then((chatResult: ChatResult) => {
                         panel.webview.postMessage({
-                            command: 'aws/chat/sendChatPrompt',
+                            command: chatRequestType.method,
                             params: chatResult,
                             tabId: message.params.tabId,
                         })
+                        chatDisposable.dispose()
+                    })
+                break
+            case quickActionRequestType.method:
+                const quickActionPartialResultToken = uuidv4()
```

Comment:
This implementation looks identical to on for `chatRequestType`.

Reply:
Given this is a test client and given we are already going to change this implementation, I don't see much point in optimizing this right now